engine/language-server/src/main/scala/org/enso/languageserver/protocol/json/JsonConnectionController.scala

```diff
@@ -259,7 +259,11 @@ class JsonConnectionController(
         receiver ! ResponseResult(
           InitProtocolConnection,
           request.id,
-          InitProtocolConnection.Result(allRoots.map(_.toContentRoot).toSet)
+          InitProtocolConnection.Result(
+            buildinfo.Info.ensoVersion,
+            buildinfo.Info.currentEdition,
```

Comment:
I'd like to repeat what I am repeating all the time: I believe we should share a _semantic protocol version_ at this moment as well. We shall increase the _semantic protocol version_ with every protocol change according to semantic versioning rules.

I hope this will happen one day @hubertp and @4e6.